pages/docs/mutation.en-US.md

```diff
@@ -74,7 +74,7 @@ function Profile () {
 
 **`revalidate`**: should the cache revalidate once the asynchronous update resolves.
 
-**`populateCache`**: should the result of the remote mutation be written to the cache.
+**`populateCache`**: should the result of the remote mutation be written to the cache. A function can be passed.
```

Comment:
```suggestion
**`populateCache`**: should the result of the remote mutation be written to the cache, or a function that receives new result and current result as arguments and returns the mutation result.
```

Reply:
Thank you! I'll add this to other languages!